assets/src/edit-story/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -684,4 +733,131 @@ describe('Right Click Menu integration', () => {
       });
     });
   });
+
+  describe('right click menu: background media', () => {
+    it("should duplicate the current page when clicking 'Duplicate page'", async () => {
```

Comment:
💭 IMO we don't need this because foreground and background media menus use the same function for 'duplicate page'. We could instead test both of them in one test together by adding this one to the other one.